### DISCUSSION TITLE
Bantu

### DISCUSSION THREAD

**Discussion by waraxeoffc** [deaafrizal/forum#56] [The Lounge]
Bang ajarin MagnifyingGlass di sesi 2 belajar buat web anime nggk muncul

![image_2024-01-17_085057666](https://github.com/deaaprizal/forum/assets/156443386/57cb74ac-6dac-4944-8ebe-ffa7e34fba7e)

**Comment by Mikael2909** [comment]
di video bang dea ada lanjutan playlist nya, mainkan absolute dan relative agar search icon nya tetap berada di dalam input search 

```
<div className="relative">
      <input
       placeholder="Cari anime"
       className="w-full p-2 rounded bg-color-primary text-color-dark"
       ref={searchRef}
       onKeyDown={handleSearch}
       />
       <button className="absolute top-2 end-2" onClick={handleSearch}>
       <MagnifyingGlass size={24} />
       </button>
    </div>
```
